.eslintrc

```diff
@@ -1,14 +0,0 @@
-{
-    "extends": "@terascope",
-    "parserOptions": {
-        "ecmaVersion": "latest",
-        "sourceType": "module"
-    },
-    "rules": {
-        "@typescript-eslint/naming-convention": "off",
-        "@typescript-eslint/no-duplicate-enum-values": "warn",
-        "import/extensions": "off",
-        "import/no-import-module-exports": "off"
```

Comment:
Are we choosing to use the defaults for `no-duplicate-enum-values` and `no-import-module-exports` or are these handled somewhere else?

Reply:
these are handled in the eslin-config library, remember these are just overrides, we have not touched the library for a very long time